copy.sh

```diff
@@ -45,7 +45,7 @@ case $lines in
         printf "$stdin" | $pbcopy
         ;;
     *)
-        filter="$(get_filter "${F:-"fzy:fzf-tmux:fzf:peco:percol:zaw"}")"
+        filter="$(get_filter "${F:-"fzf:fzy:peco:percol:zaw"}")"
```

Comment:
It's strange, I remember that I made the pull request before the second commit. Why the second commit appears here...